dipy/workflows/reconst.py

```diff
@@ -222,7 +223,7 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
             out_dir='', out_tensor='tensors.nii.gz', out_fa='fa.nii.gz',
             out_ga='ga.nii.gz', out_rgb='rgb.nii.gz', out_md='md.nii.gz',
             out_ad='ad.nii.gz', out_rd='rd.nii.gz', out_mode='mode.nii.gz',
-            out_evec='evecs.nii.gz', out_eval='evals.nii.gz'):
+            out_evec='evecs.nii.gz', out_eval='evals.nii.gz', fsl_tensor=False):
```

Comment:
The name of the parameter is misleading. Should we say nifti_standard_tensor or nifti_tensor?

Reply:
And this would be set to True per default? The problem with that is that it's not so clear what the alternative means. Here, the idea is that this tells the users that this would give them a file that interoperates with FSL. Does that make sense?

Reply:
Trying to use software names in the past did not go well. Why fsl_tensor and not trackvis_tensor or other? I think having the ```nifti_standard``` there is better and you can explain more in the docstring. And in the docstring we should have the convention explained for the choice of lower vs upper triangular and row vs column order. @ecaruyer do you agree?

Reply:
How's this? See recent https://github.com/nipy/dipy/pull/1892/commits/24dc0eaac65b9020a931aa5171a854d647234fee

Reply:
Yes, I agree with the nifti_standard suggestion. 

Just one last comment from me, I am quite sure trackvis does follow the row-order lower triangular convention, which is I think in contradiction with the docstring as it stands.

Thanks again for this PR!

Reply:
Oh yeah. I think the other software that was mentioned was Fibernavigator, but I think even that may no longer be true. Fixed in https://github.com/nipy/dipy/pull/1892/commits/162de19e392cda86a1bec4138c54c644062179ec